pkgs/by-name/oo/ooklaserver/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+, autoPatchelfHook
+}:
+let
+  filenameMapping = {
+    "x86_64-linux" = "OoklaServer-linux-x86_64-static-musl.zip";
+    # Mach-O universal binary with 2 architectures: [x86_64:Mach-O 64-bit executable x86_64] [arm64]
+    "x86_64-darwin" = "OoklaServer-macosx.zip";
+    "aarch64-darwin" = "OoklaServer-macosx.zip";
+    "aarch64-linux" = "OoklaServer-linux-aarch64-static-musl.zip";
+    "i686-windows" = "OoklaServer-windows32.zip";
+    "x86_64-windows" = "OoklaServer-windows64.zip";
```

Comment:
I believe the binary name for windows platform is different?

Reply:
ah good catch, it ends in `.exe`.

Reply:
I saw some code refer to `stdenv.isWindows` but it seems that does not exists. Any suggestion on alternative?

Reply:
There is `stdenv.hostPlatform.isWindows`, but `stdenv.hostPlatform.extensions.executable` is a better choice.